exercises/practice/dnd-character/.meta/example.go

```diff
@@ -0,0 +1,65 @@
+package dndcharacter
+
+import (
+	"math"
+	"math/rand"
+	"slices"
+)
+
+type Character struct {
+	Strength     int
+	Dexterity    int
+	Constitution int
+	Intelligence int
+	Wisdom       int
+	Charisma     int
+	Hitpoints    int
+}
+
+// Modifier should calculate the correct modifier for a given score
+func Modifier(score int) int {
+	return int(math.Floor(float64(score-10) / 2.0))
+}
+
+// Ability should generate the score for a random ability
+func Ability() int {
+	var scores []int
+
+	for i := 0; i < 4; i++ {
+		roll := RollDice()
+		scores = append(scores, roll)
+	}
+
+	return CalculateAbilityScore(scores)
+}
+
+// GenerateCharacter should return a Character with valid ability scores
```

Comment:
```suggestion
// GenerateCharacter creates a new Character with random scores for abilities 
```

The comment should also be changed in the stub given to students